packages/calcite-components/src/components/slider/slider.stories.ts

```diff
@@ -323,107 +517,261 @@ export const WithLargeFontSize_TestOnly = (): string =>
         calcite-label {
           padding: 10px;
         }
+        calcite-slider[layout="vertical"] {
```

Comment:
With the new vertical layout, the [default styles on the Storybook preview page](https://github.com/Esri/calcite-design-system/blob/b2c976214774490b5a313e1fafc67415fe212b37/packages/calcite-components/.storybook/preview-head.html#L11-L14) should be updated to reduce or eliminate the need for style overrides in individual vertical stories.